src/actions/submit/reviewers.ts

```diff
@@ -2,23 +2,29 @@ import prompts from 'prompts';
 import { KilledError } from '../../lib/errors';
 
 export async function getReviewers(args: {
-  fetchReviewers: boolean;
+  reviewers: string | undefined;
 }): Promise<string[]> {
-  if (!args.fetchReviewers) {
+  const reviewers = args.reviewers;
```

Comment:
nit: destructure in signature
```suggestion
export async function getReviewers({reviewers}: {
  reviewers: string | undefined;
}): Promise<string[]> {
```